docs/guides/08-introduction-to-image-files.md

```diff
@@ -50,7 +49,8 @@ The example below uses the `color` property to override the image fill color fro
 symbol: ramp(buckets($complaint,['Car','Bus','Building']),images([car,bus,house]))
 color: blue
 ```
-http://127.0.0.1:8080/examples/editor/index.html#eyJhIjoic3BlbmRfZGF0YSIsImIiOiIiLCJjIjoiY2FydG92bCIsImQiOiJodHRwczovL3t1c2VyfS5jYXJ0by5jb20iLCJlIjoid2lkdGg6IDcwXG5zeW1ib2w6IGNyb3NzXG5zeW1ib2xQbGFjZW1lbnQ6IGFsaWduX2NlbnRlclxuY29sb3I6IHdoaXRlIiwiZiI6eyJsbmciOjIuMTY3NjA2NDU4ODExMTc4NSwibGF0Ijo0MS40MDgwMTMzNzY3NDIyNX0sImciOjE3LjA0MjkyMzM2MTQxOTIxLCJoIjoiRGFya01hdHRlciJ9
+
```

Comment:
@davidmanzanares for the earlier comment about overriding the default color, I put that in the `Global color` sentence. Is that ok?